crates/teloxide/src/lib.rs

```diff
@@ -30,7 +30,56 @@
 //!     <img src=https://github.com/teloxide/teloxide/raw/master/media/throw-dice.gif width=420px />
 //!   </kbd>
 //! </div>
+//! 
+//! ## Working with Updates and Messages
+//! There're a great number of [update kinds](https://docs.rs/teloxide/latest/teloxide/types/enum.UpdateKind.html) and 
```

Comment:
Please outline links. Also, please use intra doc links, instead of linking `docs.rs`. e.g.

```md
There is a great number of [update kinds] and [message kinds] …

<!-- at the end of the paragraph -->

[update kinds]: crate::types::UpdateKind
[message kinds]: crate::types::MessageKind
```

(p.s. "There're" isn't a thing...)